docker/Dockerfile

```diff
@@ -109,16 +109,20 @@ RUN SCRATCH_DIR=/scratch && mkdir -p ${SCRATCH_DIR} && cd ${SCRATCH_DIR} && \
 
 # Install Kokkos
 ARG KOKKOS_VERSION=2.8.00
-ARG KOKKOS_OPTIONS="--cxxstandard=c++14 --with-serial --with-openmp --with-options=disable_deprecated_code --with-cuda --with-cuda-options=enable_lambda"
```

Comment:
Keep and pass to `cmake`

Reply:
Passing `nvcc_wrapper` as the `CMAKE_CXX_COMPILER` is going to be a pain.  I recommend omitting the path and prepending `<kokkos source dir>/bin` to the `PATH` environment variable in the `RUN` block below.